collect_app/src/main/java/org/odk/collect/android/analytics/AnalyticsEvents.java

```diff
@@ -241,7 +241,7 @@ private AnalyticsEvents() {
     public static final String ACCURACY_THRESHOLD_DEFAULT = "AccuracyThresholdDefault";
 
     /**
-     * Tracks how often form details with null or empty hashes are provided by a server
+     * Tracks how often form details with invalid hashes are provided by a server
      */
-    public static final String NULL_OR_EMPTY_FORM_HASH = "NullOrEmptyFormHash";
```

Comment:
We definitely want to stop logging this?

Reply:
I just turned it into `INVALID_FORM_HASH` but it logs the same case just with more details.

Reply:
Ach sorry I meant to delete this comment once I'd got through the review.